lib/router/mbaas.js

```diff
@@ -64,55 +64,77 @@ function initRouter(mediator, authResponseExclusionList, expressSessionMiddlewar
   });
 
   router.route('/').get(function(req, res) {
-    mediator.once('done:wfm:user:list', function(data) {
-      res.json(data);
+    mediator.request('wfm:user:list').then(function(result) {
+      if (result instanceof Error) {
+        mediator.publish('error:wfm:user:list', result);
+        return res.status(500).json({message: "Unexpected error when getting the list of users. " + result.toString()});
+      }
```

Comment:
Errors should come as a rejection (`.catch(function(error){})`) instead of in place of `result`. The `mediator.request` itself listens for the `error:` prefixed topic. see [here](https://github.com/feedhenry-raincatcher/raincatcher-mediator/blob/master/lib/mediator.js#L28)